drivers/sensor/ti_hdc/ti_hdc.c

```diff
@@ -18,8 +18,9 @@
 #define LOG_LEVEL CONFIG_SENSOR_LOG_LEVEL
```

Comment:
You may wish to add your copyright notice as you extended the functionality of the driver.